.github/workflows/release-airbyte-os.yml

```diff
@@ -0,0 +1,58 @@
+name: Release Open Source Airbyte
+on:
+  workflow_dispatch:
+    inputs:
+      partToBump:
+        description: "Please choose the type of version upgrade : major|minor|patch"
+        required: true
+        default: "patch"
+jobs:
+  releaseAirbyte:
+    runs-on: ubuntu-latest
+    environment: more-secrets
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Save Old Version
+        id: old_version
+        run: |
+          echo ::set-output name=OLD_VERSION::$(grep VERSION .env | cut -d"=" -f2)
+      - name: Release Airbyte
+        id: release_airbyte
+        env:
+          PART_TO_BUMP: ${{ github.event.inputs.partToBump }}
+          CLOUDREPO_USER: ${{ secrets.CLOUDREPO_USER }}
+          CLOUDREPO_PASSWORD: ${{ secrets.CLOUDREPO_PASSWORD }}
+        run: |
+          docker login -u airbytebot -p ${{ secrets.DOCKER_PASSWORD }}
+          ./tools/bin/release_version.sh
+      - name: Save New Version
+        id: new_version
+        run: |
+          echo ::set-output name=NEW_VERSION::$(grep VERSION .env | cut -d"=" -f2)
+      - name: Get PR Body
+        id: pr_body
+        env:
+          PREV_VERSION: ${{ steps.old_version.outputs.OLD_VERSION }}
+        run: |
+          chmod +x tools/bin/pr_body.sh
```

Comment:
we can `chmod` the `pr_body.sh` script and check that in so we don't have to do this in the workflow everytime.